data/mods/Aftershock/maps/furniture_and_terrain/terrain_spaceship.json

```
@@ -22,6 +22,29 @@
       ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_afs_augustmoon_diamond_glass",
+    "name": "diamond-matrix glass",
+    "description": "The diamond cubic lattice distorts distant light sources into eight pointed stars.  The galactic core is visible through the glass, but no spacer would ever give it more than a passing glance.",
+    "roof": "t_metal_flat_roof",
+    "symbol": "LINE_XOXO",
+    "color": "light_cyan",
+    "move_cost": 0,
+    "coverage": 100,
```

```suggestion
    "coverage": 100,
    "bash": {
      "str_min": 4000,
      "str_max": 8000,
      "sound": "whump!",
      "sound_fail": "whump!",
      "ter_set": "t_afs_augustmoon_diamond_glass"
    },
```
If you would prefer the station terrain itself to be indestructible for the time being, I've found that having a terrain break into itself accomplishes this.